lib/screens/profile_screen.dart

```diff
@@ -1,15 +1,263 @@
 import 'package:flutter/material.dart';
+import 'package:project_eureka_flutter/components/eureka_list_view.dart';
+import 'package:project_eureka_flutter/components/eureka_toggle_switch.dart';
+import 'package:project_eureka_flutter/screens/profile_onboarding.dart';
+import 'package:project_eureka_flutter/services/user_category_service.dart';
+import 'package:project_eureka_flutter/services/user_question_service.dart';
+import 'package:project_eureka_flutter/services/user_answer_service.dart';
 import 'package:project_eureka_flutter/components/side_menu.dart';
 
-class ProfileScreen extends StatelessWidget {
+class ProfileScreen extends StatefulWidget {
+  @override
+  _ProfileScreenState createState() => _ProfileScreenState();
+}
+
+class _ProfileScreenState extends State<ProfileScreen> {
+  int _role = 0;
```

Comment:
I think it would be better to rename `_role`. It took me awhile to understand that it wasn't dealing with user's role, but with the tabs, or was it?

Reply:
Good catch! It's called `_tab`_ now.